doc/docusaurus.config.ts

```diff
@@ -90,6 +90,10 @@ const config: Config = {
             {
               label: 'Nethermind',
               href: 'https://nethermind.io',
+            },
+            {
+              label: 'Discord',
+              href: 'https://discord.gg/gdpdccwQ',
```

Comment:
Is this invite link active? I tried going to it but got an `Invalid Invite` error, it could be beacuse I'm already in the server though

Reply:
![image](https://github.com/user-attachments/assets/d7586646-ff0c-4b58-83e8-e496b1fe78bd)

Interesting. Seems that the link URLs are not permanent - I'm sure I used the same method to get the link in the first place.

They can be configured to be permanent, but that may be inviting bots if the link is published.

Options:
- add a permanent link
- drop it

?

@joshklop @natebeauregard

Reply:
I think a permalink would be fine. We'll probably want a stable link for people to join and we can always enable some bot detection verification in the discord before people are admitted

Reply:
a8f9dd3